bots/tests-invoke

```diff
@@ -68,16 +71,19 @@ def main():
     return 0
 
 class PullTask(object):
-    def __init__(self, name, revision, ref, context, base=None, github_revision=None):
+    def __init__(self, name, revision, ref, context, base, remote, github_revision):
+    def __init__(self, name, revision, ref, context, base, remote, github_revision, updater):
```

Comment:
This duplicates the `__init__()` definition. This cases all bots to break:
```
+ bots/tests-invoke --rebase=master verify/ubuntu-1604 pull/8576/head

  File "bots/tests-invoke", line 75

    def __init__(self, name, revision, ref, context, base, remote, github_revision, updater):

      ^

IndentationError: expected an indented block
```

This breaks the bots, so I'm down-prioritizing this PR so that others can get tested.

Reply:
Whoa, care-less conflict resolution on my part, sorry.